### PR TITLE
Update skeletons.md

### PR DESCRIPTION
**Pull request by AlexITC**
Add the wiringbits scala-webapp-template, and reorganize the existing links, some are outdated and others are archived, let's reflect that in the docs.

### CODE REVIEW

File: libraries/skeletons.md [hunk 1]
```diff
@@ -19,28 +38,28 @@ It is made with Scala.js & [Play Framework](https://www.playframework.com/). App
...
+Allows to create [Google custom functions](https://developers.google.com/apps-script/guides/sheets/functions) in [Google SpreadSheets](https://www.google.com/intl/en_UK/sheets/about/) with Scala.
 
-### [AWS Lambda with Scala.js](https://github.com/bgahagan/scalajs-lambda.g8)
-A skeleton to build an AWS Lambda handler using Scala.js 1.0. Uses sbt-scalajs-bundler to manage npm dependencies and sbt-native-packager to package lambda for deployment.
 
-### [npm scalajs-starter](https://github.com/konradst/scalajs-starter)
-Scala.js Node.js skeleton app / library handled with ```npm start```, ```build```, ```test```, ```clean``` commands.
+## Archived
+
+These applications are not maintained anymore ubt could be useful to keep them as references.
```
Comment (sjrd) [inline libraries/skeletons.md:50]:
```suggestion
These applications are not maintained anymore but could be useful as references.
```

File: libraries/skeletons.md [hunk 2]
```diff
@@ -19,28 +38,28 @@ It is made with Scala.js & [Play Framework](https://www.playframework.com/). App
...
+
 Project generator for the Udash framework - creates a customized, complete and runnable Udash based application.
 
-### [Scala Google Spreadsheets](https://github.com/sherpal/Scala-Google-Spreadsheets)
-Allows to create [Google custom functions](https://developers.google.com/apps-script/guides/sheets/functions) in [Google SpreadSheets](https://www.google.com/intl/en_UK/sheets/about/) with Scala.
+
+
+
+
+
```
Comment (sjrd) [inline libraries/skeletons.md:65]: Blank lines ;)

File: libraries/skeletons.md [hunk 3]
```diff
@@ -6,6 +6,25 @@ title: Project Skeletons
...
+Example application showing how you can integrate [Play](https://www.playframework.com/) with Scala.js.
+
+### [AWS Lambda with Scala.js](https://github.com/bgahagan/scalajs-lambda.g8)
+
+A skeleton to build an AWS Lambda handler using Scala.js 1.0. Uses sbt-scalajs-bundler to manage npm dependencies and sbt-native-packager to package lambda for deployment.
+
+### [npm scalajs-starter](https://github.com/konradst/scalajs-starter)
+
+Scala.js Node.js skeleton app / library handled with ```npm start```, ```build```, ```test```, ```clean``` commands.
```
Comment (sjrd) [inline libraries/skeletons.md:19]: This one seems outdated. It uses sbt-scalajs 1.0.0-M8.

**Review by sjrd — CHANGES_REQUESTED** [review]

File: libraries/skeletons.md [hunk 4]
```diff
@@ -19,28 +38,28 @@ It is made with Scala.js & [Play Framework](https://www.playframework.com/). App
...
+Proof of concept to determine if Scala.js could be leveraged to make a Node.js module.
 
-### [scalaWUI](https://github.com/mathieuleclaire/scalaWUI)
-A ready to work Client/Server application built with [scalatra](http://scalatra.org/),
-[scala.rx](https://github.com/lihaoyi/scala.rx), [autowire](https://github.com/lihaoyi/autowire). An example using the
-D3.js lib.
 
 ### [Udash Generator](http://guide.udash.io/#/bootstrapping/generators)
+
 Project generator for the Udash framework - creates a customized, complete and runnable Udash based application.
```
[the remark this answers is not in the document]
Comment (AlexITC) [reply libraries/skeletons.md:59]: Udash-generator (which is the linked project) is archived: https://github.com/UdashFramework/udash-generator/

File: libraries/skeletons.md (hunk 3 above)
Comment (AlexITC) [reply libraries/skeletons.md:19]: Indeed, I was keeping everything related to 1.0.0 as recent, I'll update the section

File: libraries/skeletons.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (AlexITC) [reply libraries/skeletons.md:59]: Anyway, it is up to you what I do with this one

**Review by sjrd — COMMENTED** [review]
Thanks. Could you squash the two commits together, please?

**Comment by AlexITC** [comment]
Done, FYI: Github has the `Squash and Merge` option to do this

**Review by sjrd — APPROVED** [review]

**Comment by sjrd** [comment]
> Done, FYI: Github has the `Squash and Merge` option to do this

Yes, I know, but that doesn't create a Merge commit at all. I want a "Squash and create a Merge commit" option.

**Pull request merged**